tests/tensor/test_subtensor.py

```diff
@@ -771,8 +771,16 @@ def test_grad_2d_inc_set_subtensor(self):
 
                 t = op(n[:z, :z], m)
                 gn, gm = pytensor.grad(pt_sum(t), [n, m])
-                utt.verify_grad(lambda m: op(n[:z, :z], m), [mv], mode=self.mode)
-                utt.verify_grad(lambda nn: op(nn[:z, :z], mv), [data], mode=self.mode)
+                utt.verify_grad(
+                    lambda m: op(n[:z, :z], m),
+                    [mv],
+                    mode=self.mode,
```

Comment:
If you remove the commas pre-commit will go back to the previous formatting and avoid these unrelated changes

```suggestion
                    mode=self.mode
```